mlrun/run.py

```diff
@@ -309,7 +309,7 @@ def load_func_code(command="", workdir=None, secrets=None, name="name"):
                     command = temp_file.name
                     temp_file.write(code)
 
-        elif command and not is_remote:
+        elif command and "http://" not in command and not is_remote:
```

Comment:
add a comment explaining why would (or what would) http is part of command and also the why you skip it.
I think also worth skipping "https"